common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-tagsAreTopLevelPropertiesOnly_2024-06-13-22-19.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Add TagsAreNotAllowedForProxyResources rule to check if tags are defined as top-level property, not in the properties bag",
```

Comment:
>tags are defined as top-level property, not in the properties bag [](http://example.com/codeflow?start=74&length=65)

needs to be updated too.

Reply:
Removing this file as per Konrad's suggestion